browser/src/control/jsdialog/Widget.Combobox.js

```diff
@@ -237,11 +237,25 @@ JSDialog.combobox = function (parentContainer, data, builder) {
 
 		resetSelection();
 		for (var i in entries) {
-			if (entries[i] == this.value) {
+			if (entries[i].text == this.value) {
```

Comment:
we need more typescript :)

Reply:
this is complicated... I think the "JS-generated entries" have text prop (eg. used for dropdown) but the "core-generated entries" come without it (see example JSON in the header and core impl: https://opengrok.libreoffice.org/xref/core/vcl/source/control/combobox.cxx?r=4b53712c#1570
where it has simple value.

we need to handle both cases

`entries[i] == this.value || entries[i].text == this.value"